File: app/src/main/java/com/example/carbonmovies/api/Service.java
package com.example.carbonmovies.api;

import com.example.carbonmovies.model.MoviesResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface Service {

    @GET("movie/popular")
    Call <MoviesResponse> getPopularMovies(@Query("api_key") String apiKey);

    @GET("movie/top_rated")
    Call <MoviesResponse> getTopMovies(@Query("api_key") String apiKey);
}

File: app/src/main/java/com/mindinitiatives/carbonmovies/api/Service.kt
package com.mindinitiatives.carbonmovies.api

import com.mindinitiatives.carbonmovies.model.MoviesResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface Service {
    @GET("movie/popular")
    fun getPopularMovies(@Query("api_key") apiKey: String?): Call<MoviesResponse?>?
    @GET("movie/top_rated")
    fun getTopMovies(@Query("api_key") apiKey: String?): Call<MoviesResponse?>?
}